File: redsheet-portal/src/app/database-models/v-project-stakeholder.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vProjectStakeholder {
    constructor(vProjectStakeholder?: vProjectStakeholder )  {
        let properties = [ "actionRequried", "className", "communicationModeName", "frequancy", "iconClassName", "projectModuleId", "projectStakeholderId", "stakeholderName",];
        for (let property of properties)
            if (vProjectStakeholder && vProjectStakeholder[property])
                this[property] = vProjectStakeholder[property];
    }
 
    @required()
    @maxLength(200)
	actionRequried : string =   undefined;
 
    @required()
    @maxLength(200)
	className : string =   undefined;
 
	communicationModeName : string =   undefined;
 
    @required()
    @maxLength(250)
	frequancy : string =   undefined;
 
	iconClassName : string =   undefined;
 
	projectModuleId : number =   0 ;
 
    @range(1,2147483647)
	projectStakeholderId : number =   undefined;
 
    @required()
    @maxLength(100)
	stakeholderName : string =   undefined;


}

File: RedSheet.DbEntities/Models/vProjectStakeholder.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vProjectStakeholders", Schema = "dbo")]
	[Serializable]
	public partial class vProjectStakeholder  
	{
 
	[Required] 
	[MaxLength(200)]
	public string ActionRequried { get; set; } 
 
	[Required] 
	[MaxLength(200)]
	public string ClassName { get; set; } 

	public string CommunicationModeName { get; set; } 
 
	[Required] 
	[MaxLength(250)]
	public string Frequancy { get; set; } 

	public string IconClassName { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int ProjectModuleId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ProjectStakeholderId { get; set; } 
 
	[Required] 
	[MaxLength(100)]
	public string StakeholderName { get; set; } 

	public vProjectStakeholder()
	{
    }

    }
}

